ci/licenses_golden/licenses_flutter

```diff
@@ -1,6 +1,39 @@
 UNUSED LICENSES:
 
+====================================================================================================
```

Comment:
Paging @Hixie for review of licensing concerns.

Reply:
I chatted with Ian last week. He's OOO this week, but this is the feedback:
1. Move the code to third_party
2. Use the Skia license

Reply:
Ok. Sounds fine.